### PR TITLE
Modify TiFlash's troubleshooting procedure to adapt current version

### PR DESCRIPTION
**Pull request by CalvinNeo**
Signed-off-by: CalvinNeo <calvinneo1995@gmail.com>

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

TiFlash has refactored some modules since 6.0, so we need to update some troubleshooting procedures.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v6.0 (TiDB 6.0 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by shichun-0415 — APPROVED** [review]
LGTM

**Review by flowbehappy — APPROVED** [review]

File: tiflash/troubleshoot-tiflash.md [hunk 1]
```diff
@@ -179,44 +179,23 @@ show warnings;
...
-      - 检查节点间能否相互连通。
-
-    > **注意：**
-    >
-    > 如问题依然无法解决，收集相关组件的日志进行排查。
-
-6. 检查表是否创建 `placement-rule`。
-
-    检查 `flash_cluster_manager.log` 日志，查找是否存在关键字 `Set placement rule … table-<table_id>-r`。
+    搜索 TiDB DDL Owner 的日志，检查 TiDB 是否通知 PD 添加 `placement-rule`。对于非分区表搜索 `ConfigureTiFlashPDForTable`；对于分区表，搜索 `ConfigureTiFlashPDForPartitions`。
```
Comment (JaySon-Huang) [inline tiflash/troubleshoot-tiflash.md:184]: 怎么获取哪一个 tidb 实例是 tidb ddl owner 呢？

File: tiflash/troubleshoot-tiflash.md [hunk 2]
```diff
@@ -229,31 +208,22 @@ show warnings;
...
-    在 `flash_cluster_manager.log` 中，检查同步卡住的表对应的 `flash_region_count` 是否有更新。
+2. 检查是否有 `down peer` （`down peer` 没有清理干净可能会导致同步卡住）。
 
-    - 没有更新，检查下一步。
-    - 有更新，检查是否有 `down peer` （`down peer` 没有清理干净可能会导致同步卡住）。
-
-      - 执行 `pd-ctl region check-down-peer` 命令检查是否有 `down peer`。
-      - 如果存在 `down peer`，执行 `pd-ctl operator add remove-peer\<region-id> \<tiflash-store-id>` 命令将其清除。
+    - 执行 `pd-ctl region check-down-peer` 命令检查是否有 `down peer`。
+    - 如果存在 `down peer`，执行 `pd-ctl operator add remove-peer\<region-id> \<tiflash-store-id>` 命令将其清除。
```
Comment (JaySon-Huang) [inline tiflash/troubleshoot-tiflash.md:214]:
```suggestion
    - 如果存在 `down peer`，执行 `pd-ctl operator add remove-peer <region-id> <tiflash-store-id>` 命令将其清除。
```
这里应该不需要 `\` 字符？

File: tiflash/troubleshoot-tiflash.md [hunk 3]
```diff
@@ -229,31 +208,22 @@ show warnings;
...
 
 3. 查看 CPU 使用率。
 
     在 Grafana 界面，选择 **TiFlash-Proxy-Details** > **Thread CPU** > **Region task worker pre-handle/generate snapshot CPU**，查看监控中 `<instance-ip>:<instance-port>-region-worker` 对应线程的 CPU 使用率。
 
-    若曲线为一条直线，表示 TiFlash 发生卡死，可强制杀进程重启或联系 PingCAP 技术支持人员。
```
Comment (JaySon-Huang) [inline tiflash/troubleshoot-tiflash.md:246]: 保留这句？ "若曲线为一条直线，表示 TiFlash 内部发生卡死，可尝试强制杀进程重启"

Comment (flowbehappy) [reply tiflash/troubleshoot-tiflash.md:246]: 似乎没有出现过类似情况？

Comment (JaySon-Huang) [reply tiflash/troubleshoot-tiflash.md:246]: 早期的 4.0.x/5.0.x 版本，DDL 和 Raft 线程之间发生死锁问题，Raft 线程被全部 block 的时候，可能会有这样的现象

Comment (flowbehappy) [reply tiflash/troubleshoot-tiflash.md:246]: 这是给 6.1 的，所以可以不用写了？

File: tiflash/troubleshoot-tiflash.md (hunk 1 above)
Comment (CalvinNeo) [reply tiflash/troubleshoot-tiflash.md:184]: 目前是从 admin show ddl 中获取，这个方案是实时的，但可能不是一个比较直接的方案。[直接一点的方案是提供一个 tidb_ddl_owner()](https://github.com/pingcap/tidb/issues/10024)。
因此，我觉得最好不要在这篇 TiFlash 的文档中介绍 TiDB 的一些方案。

File: tiflash/troubleshoot-tiflash.md (hunk 3 above)
Comment (JaySon-Huang) [reply tiflash/troubleshoot-tiflash.md:246]: 那么 "3. 查看 CPU 使用率" 这一步，查看完之后做什么呢？或者把这一步都去除掉？

File: tiflash/troubleshoot-tiflash.md (hunk 1 above)
Comment (JaySon-Huang) [reply tiflash/troubleshoot-tiflash.md:184]: ok

File: tiflash/troubleshoot-tiflash.md (hunk 3 above)
Comment (shichun-0415) [reply tiflash/troubleshoot-tiflash.md:246]: @CalvinNeo PTAL

**Review by JaySon-Huang — APPROVED** [review]
LGTM

**Pull request merged**